### DISCUSSION TITLE
美國人用自駕車來配送食物！

### DISCUSSION THREAD

**Discussion by qeeeee39** [1101-GS3073-C/Python-Community-GS3073-C#222] [General]
在美國，已經在使用自駕車來做食物的外送服務，而未來幾年外送自駕車的數量將會越來越多。

這個食物外送自駕車和我們常聽到的tesla、apple自駕車相比簡單了許多，因為他只需要在人行道上移動。

台灣和美國相比下來，我們好慢啊
我們現在還在foodpanda，別人都L4純電自駕車了

https://technews.tw/2021/12/24/self-driving-delivery-service/

**Comment by lx3211** [comment]
感覺只在人行道移動的話可以避免很多交通事故的麻煩

**Comment by brian431** [comment]
不知道這種無人監督的小型自駕車會不會在路上一直被拍打破壞等等

**Comment by NcuJerryKao** [comment]
用無人機 無人機送餐
若能發展無人機自駕外送 那送餐便可更高效率 而且不會占用交通要道

**Comment by BingRong1217** [comment]
感覺是很好的發展方向

**Comment by Yilin20021016** [comment]
但是會有不少安全問題吧 現在自駕車的意外不算少

**Comment by HsuChen88** [comment]
在人行道上馳騁的話，或許會需要一個外送專用道

**Comment by skyling0299** [comment]
外送專用道vs腳踏車專用道vs行人專用道哈哈
讓我想到一篇梗圖說叫直升機uber比叫車輛的便宜

**Comment by skyling0299** [comment]
歐特明電子與國網中心合作，開發「自動代客泊車系統(AVP)」，以 AI 影像辨識技術為基礎，利用汽車的 3D 環景攝影機所收錄的影像，搭配超音波雷達，並透過 AI 深度學習技術，在進入停車場到記憶好的停車格約 100 公尺範圍內，不須駕駛在旁，即可在車內無人的情況下精準的把車子駛入停車格。

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
如果有人偷走餐點的話，後面引發的安全問題也不容小覷

**Comment by awehao** [comment]
我覺得沒料，像有人說得道路安全問題，還有食物能否不被他人取走，不如讓變形金剛來送好了

**Comment by Raywei0214** [comment]
送餐的初始想法就大有問題!!foodpanda跟uber都送不好了，還指望變數更大的機器，想當年我第一次訂uber多等了1小時，菜都涼了，一整天的心情也涼了，那個外送員的評價也涼了

**Comment by awkward-willy** [comment]
這樣的技術能快速且顯著的改變我們的生活，不過我也相當好奇，若真的實行，自駕車在運送過程中若發生交通事故，是否有辦法及時呼叫其他自駕車前往支援或聯繫人員處理。同時，若發生交通事故，責任的歸屬會落在何方，尤其是當雙方都是自駕車時，會不會出現自己究責自己的情況。

**Comment by RogelioKG** [comment]
我覺得不太行，在人行道上的話會和行人爭道，
而且確實也難防惡意破壞，到時候就真的是在地美食了。
如果是在目前的台灣實現的話應該更難，畢竟我們的人行道是走到一半會斷掉的那種，
或者走到一半只剩90公分寬，然後被一台挖土機和隨便放置的幾個三角錐給擋住去路。

**Comment by boson13579** [comment]
雖然外送的自駕車是小台，且在人行道上運行，但不可避免的是我們還是要考慮到自駕車可能軌跡偏移，開到馬路上威脅馬路駕駛者的安全。因此，在台灣，至少也要等相關法令通過、多次道路測試才能真正上路。
